### PR TITLE
ignore whitespace in case there are params

### LINKED ISSUE #108: Params with whitespace error
While setting params with whitespace I get the following error:

```
[error] java.lang.RuntimeException: 'minikube version' is not a valid environment variable name
```
Code example:

```
githubWorkflowAddedJobs := Seq(WorkflowJob(
    id = "integration-kubernetes-v1-22",
    name = "integration-kubernetes-v1-22",
    steps = List(
      WorkflowStep.Checkout,
      WorkflowStep.Use(
        ref = UseRef.Docker(image = "manusa/actions-setup-minikube", tag = "v2.4.3"),
        params = Map(
         "minikube version" -> "v1.23.2",
         "kubernetes version" -> "v1.22.1",
         "github token" -> "${{ secrets.GITHUB_TOKEN }}")),
      WorkflowStep.Sbt(
        List("It/test")
      )
    )
  ))
  ```
  
  Github actions example
  https://github.com/hagay3/skuber/runs/6249580109?check_suite_focus=true

### PR DESCRIPTION
**Pull request by hagay3**
Resolves #108

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,5 @@ target/
 tags
 
 .bsp
+
+.idea
```
Comment (mdedetrich) [inline .gitignore:11]: Can you revert this change? You shouldn't be adding IDE specific files to a projects gitignore (instead users should be using a global gitignore on their machine, see https://stackoverflow.com/a/7335487

**Review by mdedetrich — CHANGES_REQUESTED** [review]
@hagay3 Sorry for taking so long on this. I have some minor nit and once some other PR's are merged I will look into this to get it merged into main

**Comment by hagay3** [comment]
> @hagay3 Sorry for taking so long on this. I have some minor nit and once some other PR's are merged I will look into this to get it merged into main

@mdedetrich fixed.

File: .gitignore (hunk 1 above)
Comment (hagay3) [reply .gitignore]: > Can you revert this change? You shouldn't be adding IDE specific files to a projects gitignore (instead users should be using a global gitignore on their machine, see https://stackoverflow.com/a/7335487

fixed.

Comment (hagay3) [reply .gitignore]: @mdedetrich ^

**Review by mdedetrich — APPROVED** [review]
@hagay3 Thanks for taking the time to rebase the PR, lgtm
